docs/articles/documentation/using-testcafe/programming-interface/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: TestCafe Programming Interface
+permalink: /documentation/using-testcafe/programming-interface/
+---
+# TestCafe Programming Interface
+
+This section provides reference for the API that allows you use TestCafe from your NodeJS applications.
+
+## Summary
```

Comment:
Don't think so.
We have two parts here: a brief *overview* like a 'getting-started' and *reference*.
And they are two different parts.
So, you suggest that the overview is the introduction to the reference...